common/filestore_utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for interacting with the file storage."""
+
+from common import logs
+from common import experiment_utils
+
+logger = logs.Logger('filestore_utils')
+
+
+def _using_gsutil():
+    """ Test whether using gsutil format paths. """
+    try:
+        experiment_path_format = experiment_utils.get_cloud_experiment_path()
+    except KeyError:
+        experiment_path_format = 'gs://default'
+
+    return experiment_path_format.startswith('gs://')
+
+
+if _using_gsutil():
+    from common import gsutil as filestore_utils_impl
+else:
+    # When gsutil is not used in the context,
```

Comment:
Unless youre trying to format a list, try to fit as much on each comment line as possible (without going over limit).
(I think `here` and `it` probably fit on this line.